Golem.Tests/Golem.IntegrationTests.csproj

```diff
@@ -15,6 +15,7 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="8.0.0" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.7.1" />
     <PackageReference Include="Moq" Version="4.20.70" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
Any particular reason to use Newtonsoft.Json over System.Text.Json? From what I remember it is usually slower and heavier

Reply:
I wanted to get gpu price from Json in least verbose way possible:
`presets_obj.SelectToken("$.presets[?(@.name == 'dummy_copy')]")["usage-coeffs"]["golem.usage.gpu-sec"];`
Not sure if `System.Text.Json` allows for such queries.

Reply:
But, yes, probably it would be better to not add extra library. I will look into ` System.Text.Json` linq.

Reply:
Sorry, `System.Text.Json` seems to reuiqre parsing Json element by element, I am not familiar withs its API, and the code I have already works.